aospy/automate.py

```diff
@@ -237,8 +238,30 @@ def _combine_core_aux_specs(self):
 
     def create_calcs(self):
         """Generate a Calc object for each requested parameter combination."""
-        return [Calc(CalcInterface(**sp)) for sp in
-                self._combine_core_aux_specs()]
+        specs = self._combine_core_aux_specs()
+
+        for spec in specs:
+            spec['dtype_out_time'] = _prune_invalid_time_reductions(spec)
```

Comment:
is spec['dtype_out_time'] always a tuple?

Reply:
I think it's reasonably safe to say here that it will be an iterable of some kind; the documentation states that a list is required in the constructor of a `CalcSuite`:
http://aospy.readthedocs.io/en/latest/api.html?highlight=calcsuite#aospy.automate.submit_mult_calcs.